client/landing/gutenboarding/onboarding-block/vertical-select/style.scss

```diff
@@ -1,6 +1,14 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 
 .vertical-select {
+	width: 400px;
+}
+
+.vertical-select__suggestions {
+	width: inherit;
+	max-width: inherit;
+	margin-left: 0.2em;
```

Comment:
Supposedly this is the 8px left margin removed from below. `0.2em` is the margin used in question component and updated here to match 🤔  https://github.com/Automattic/wp-calypso/blob/207ba00b4a818ac13cf2d32a23374556ca81aa5d/client/landing/gutenboarding/onboarding-block/question/style.scss#L32.